### PR TITLE
Add Express Gateway image

### PR DESCRIPTION
**Pull request by XVincentX**
Add [Express Gateway](https://express-gateway.io) official image to charts

### CODE REVIEW

**Comment by XVincentX** [comment]
CLA signed.

File: stable/express-gateway/README.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+```
+
+The command removes all the Kubernetes components associated with the
+chart and deletes the release.
+
+## Configuration
+
+### General Deployment Configuration Parameters
+
+The following table lists the configurable parameters of the Kong chart
```
Comment (davidkarlsen) [inline stable/express-gateway/README.md]: it mentions Kong - but should it be Express Gateway  - there seems to be more references to Kong as well

Comment (XVincentX) [reply stable/express-gateway/README.md]: Correct, I've been using their Readme as a baseline. I'll fix these right now.

**Comment by XVincentX** [comment]
@mattfarina @davidkarlsen 

Anything else I should be doing?

**Review by mattfarina — COMMENTED** [review]
This looks great. Can you please add a `.helmignore` file? See https://github.com/helm/charts/blob/master/stable/minio/.helmignore for an example.

**Comment by XVincentX** [comment]
@mattfarina Added! 👍

**Comment by XVincentX** [comment]
@mattfarina Thanks a lot, this is a great news. Do you know how much time is needed to see it propagated in the official repository https://kubernetes-charts.storage.googleapis.com?

**Comment by mattfarina** [comment]
@XVincentX It can take up to 30 some minutes before the jobs pulls it in. If it's longer than than please let us know.

**Comment by XVincentX** [comment]
Thanks, I was able to correctly see the Chart in the catalog. This is great.

I've requested an update in #7255

**Pull request merged**